src/pages/STG_PanelOppNamingBatch.page

```diff
@@ -12,7 +12,7 @@
                 strBatchComponentLabel="{!$Label.stgLabelOppNamingRefreshTitle}"
                 cNumberOfJobs="1"
                 startPolling="True"
-                stopPollingOnComplete="True"
+                stopPollingOnComplete="False"
```

Comment:
@RadSFDeveloper @force2b I changed this to false so that it'll continue to poll for the job even after the job is complete, so as batch jobs chain, the latest batch job gets displayed. For example, let's say there 15,000,000 Opportunities in an org.

* When the job start, the progress container will say something along the lines of "1 out of 5000 batches processed", which is the first 10,000,000 Opportunities.
* When that batch is done, it will chain another job for the remaining 5,000,000 opportunities
* The progress container will then be replaced by this next job, saying something along the lines of "1 out of 2500 batches processed".

I imagine a better scenario would be to display several progress containers so you can see the progress of the chain, though with this being a P1 bug, maybe that should go into another work item so that this fix can go out sooner?

Reply:
I don't see it being a major issue, though if I'm not mistaken the way that component works is that it gets the latest batch job running and displays its status - not specifically the job that was executed. It's possible that the component may pick up a different job after this one completes, even if that job isn't the same OppNaming job. 

The only other thing I'd note is that I would very surprised if an admin in a large org (one with more than 10M Opps) would actually stay on the NPSP Settings page for that many hours to monitor progress. If that's true, then I'd say we don't need to modify the component.

Reply:
Ah good point on the it getting the latest batch job. You are probably right about admins actually staying on the page, I'll revert this change. If it does become an issue later, we can address this with another work item